app/routes/how-can-i-help.community.tsx

```diff
@@ -1,14 +1,141 @@
 import {MetaFunction} from '@remix-run/node'
+import DropDown from '~/components/DropDown'
+import HelpItem from '~/components/HowCanIHelp/HelpItem'
+import CategoryCarousel from '~/components/HowCanIHelp/CatgoryCarousel'
 import Base from '~/components/HowCanIHelp/Base'
+import {useEffect} from 'react'
+import VideoImg from '~/assets/video_thumbnail.png'
+import PlayIcon from '~/components/icons-generated/Play'
 
 export const meta: MetaFunction = () => {
   return [{title: 'How Can I Help? - AISafety.info'}]
 }
 
+const InPerson = () => (
+  <>
+    <div className="padding-bottom-24">
+      <div className="flexbox padding-bottom-56">
+        <h2 className="teal-500 padding-bottom-40" id="new-to-aisafety">
+          In person communities
+        </h2>
+        <p className="grey default">
```

Comment:
To make it just half the page, you would apply the  col-6 class to this p

Reply:
Yep, that worked